inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -386,18 +386,25 @@ public void deleteConcept(KnowledgeBase kb, KBConcept aConcept)
     @Override
     public KBHandle createProperty(KnowledgeBase kb, KBProperty aProperty)
     {
-        if (StringUtils.isNotEmpty(aProperty.getIdentifier())) {
+        if (StringUtils.isNotEmpty(aProperty.getIdentifier())
+                && (!aProperty.getIdentifier().equals(kb.getSubclassIri().stringValue()))) {
```

Comment:
What is happening here? The exception message doesn't seem to match the condition. I also do not understand the condition - there is no comment explaining what happens.

Reply:
We do not need any extra message here as `!aProperty.getIdentifier().equals(kb.getSubclassIri().stringValue())` is added as IF condition, to avoid a only check of `StringUtils.isNotEmpty(aProperty.getIdentifier()`. This will only happen when we create subclass.